evm/deploy-contract.js

```diff
@@ -171,6 +284,12 @@ program.addOption(
 program.addOption(new Option('-a, --artifactPath <artifactPath>', 'artifact path').makeOptionMandatory(true));
 program.addOption(new Option('-c, --contractName <contractName>', 'contract name').makeOptionMandatory(true));
 program.addOption(new Option('-n, --chainNames <chainNames>', 'chain names').makeOptionMandatory(true));
+program.addOption(
+    new Option('-m, --deployMethod <deployMethod>', 'deployment method')
+        .choices(['create', 'create2', 'create3'])
+        .default('create3')
```

Comment:
```suggestion
        .default('create2')
```
let's default to create2. create3 can be used incorrectly easily.